tests/x11/firefox_audio.pm

```diff
@@ -33,6 +33,11 @@ sub run {
             last if (match_has_tag 'test-firefox_audio-1');
         }
     }
+    if (match_has_tag 'test-firefox_audio-notplayed') {
+        record_info('poo#186585', 'Play the audio file manually');
+        assert_and_click('test-firefox_audio-notplayed');
+        wait_still_screen 2;
```

Comment:
I assume this whole code section is only intended as a temporary workaround until you could investigate teh problem further?